file.go

```diff
@@ -1,37 +1,115 @@
-package jcsv
+package main
```

Comment:
Why have you changed the package `jcsv` to `main`?

Reply:
GoLand editor was prompting an error for not finding `main` package.

> Why have you changed the package `jcsv` to `main`?

Reply:
That is because you have a `main()` function to execute. `main()` function should be in the `main` package.
Don't add `main()` in the package `jscv`.